mehl.cloud.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Csl. Thomas Lüchow
+status: Teacher and Coach for Adults
```

Comment:
```suggestion
name: mehl.cloud
status: OK
```

This is the name of the instance, not of the person - I'd use mehl.cloud. Also because the name is a double with daleth.cafe otherwise.

The name is what's shown on the start page of https://providers.delta.chat/

That status needs to be `OK`, so we can show in the app that the server is spelled correctly.